Documentation/devel/quickstart-dev.md

```diff
@@ -0,0 +1,92 @@
+# Preparing development environment and first rkt build
+This is an example configuration for fresh installation of Ubuntu 16.04 GNOME.
+This doc is also a short quick start guide. For a detailed reference see [hacking.md](https://github.com/coreos/rkt/blob/master/Documentation/hacking.md)
+### Required software
+On fresh system install few software packages to correctly build rkt:
+```sh
+$ wget -qO- https://raw.githubusercontent.com/coreos/rkt/master/scripts/install-deps-debian-sid.sh | sudo sh
```

Comment:
Also same script is in cloned later repository, so installation of dependencies other than git - could be done by running local script, after this clone.